lib/tarfiles.py

```diff
@@ -73,10 +73,9 @@ def check_we_can_write() -> None:
 
 def tarchmod(tfn: str) -> str:
     """copy a tarfile to a compressed tarfile changing modes of contents to 755"""
-    os.environ["GZIP"] = "-n"
-    ofn = os.path.basename(f"{tfn}.o.gz")
+    ofn = os.path.basename(f"{tfn}.tbz2")
```

Comment:
BTW - do we know that RCDS plays nicely with tbz2 files?  I don't see why it wouldn't, but the integration tests went OK with this change?

Reply:
Besides wanting to double-check this, this PR LGTM.

Reply:
I'll rerun them. I can't recall now if I last ran them before or after this change.. sigh.

Reply:
(I mean , with the suffix tbz2;  it's done the gzipped tarballs already...)

Reply:
Did these tests work?  If so, I'll go ahead and merge.

Reply:
Sigh... My tests on friday got stuck on a job that just wouldn't come out of Idle... I'm re-running.

Reply:
Okay, found one bug so far... Still need to figure out what test is having the other problem...  It's making a very funky dag submission which keeps getting restarted and exiting 255 in the joblog; and doesn't seem to have the dag_wrapper.sh set as the executable, and.?!?!?